sample/build.gradle

```diff
@@ -18,5 +18,5 @@ dependencies {
     compile project(':rxpermissions')
 
     compile rootProject.ext.appCompat
-    compile 'com.jakewharton.rxbinding:rxbinding:0.4.0'
+    compile 'io.reactivex.rxjava2:rxandroid:2.0.0-RC1'
```

Comment:
Yup I know but we can't use version 0.4.0 which only supports RxJava 1. However the MainThreadDisposable is needed in RxView2 which we only get from RxAndroid